cpp/include/cuml/experimental/fil/treelite_importer.hpp

```diff
@@ -331,6 +331,64 @@ struct treelite_importer {
     return result;
   }
 
+  // This function returns a modified copy of a given Treelite model if it contains
+  // at least one degenerate tree. If the model contains no degenerate tree, then
+  // the function returns nullptr.
```

Comment:
Can we do doxygen style docs as this function is part of the public API?

Reply:
How do we mark this as a private API. This isn't meant to be consumed by the end user.

Reply:
Anonymous namespace will keep this in internal linkage, and add it in `detail` folder to be even safer.